packages/lesswrong/components/forumEvents/GivingSeason2024Banner.tsx

```diff
@@ -20,6 +20,14 @@ import type { ForumIconName } from "../common/ForumIcon";
 
 const DOT_SIZE = 12;
 
+/** Style to pass pointer events through one element, but not have this affect all children */
+const PASS_THROUGH_POINTER_EVENTS = {
+  pointerEvents: "none",
+  "& > *": {
+    pointerEvents: "all",
+  },
+}
+
```

Comment:
One of the trickier parts of this was making it so that you can't place a heart on elements covering part of the banner (so you can still use these elements). This required setting all the z indexes the same and carefully managing which elements capture pointer events

Reply:
Ah, well seeing as this is working now I'm going to leave it as is, but this is a good idea for if managing this pass-through thing becomes too complicated (e.g. when we want to use this for a non-giving-season banner)